admin/data/sql/migrations/054.do.update-admin-passwords.js

```diff
@@ -1,7 +0,0 @@
-'use strict'
```

Comment:
@danwilliams what about the undo script?

Reply:
good catch @GeorgeChatzigiannis